getting-started/organizations.mdx

```diff
@@ -0,0 +1,56 @@
+---
+title: "Organization options"
+---
+
+PropelAuth allows your users to create and manage [organizations](/overview/organizations).
+If you are building a B2B product, organizations are the companies using your product.
+
+![an organization management page](/images/organizations-2.png)
+
+If you aren't ready to add organizations to your product yet, you can enable/disable them at any time.
+PropelAuth provides a number of options for configuring your users organization experience, including changing the name itself.
+
+## Organization options
+
+### All users must be in an organization
+
+This setting will make sure that all users must be in at least one organization before they are allowed to use your product.
+
+The signup/login flows will direct the user to create or join an organization.
+
+### Users can create their own organizations
+
+If enabled, users have access to the 'Create Org' UI, allowing them to create their own organizations.
+
+Disable this if you want to manually create organizations yourself, on behalf of your users.
+
+### Maximum number of orgs per user
+
+This sets the limit on the number of organizations a user can be in at once.
+Most commonly, we see people setting this to either 1 (meaning each user can only be in one organization) or "as many as they want".
+
+### Changing the word "Organization"
+
+If you use a different word for organizations (maybe team, company, tenant, workspace, group, or something domain specific),
+you can configure this by changing the organization metaname.
+
+![changing the org metaname](/images/org-metaname.png)
```

Comment:
Possibly adding a screenshot here that just shows that you can name orgs whatever you want? You mention it in the passage above but maybe one of the example screenshots can say Team and one can say something more custom?